packages/webpack/test/snapshots/e2e-scuttle.spec.js.md

```diff
@@ -0,0 +1,16 @@
+# Snapshot report for `test/e2e-scuttle.spec.js`
```

Comment:
The only snapshot we might need for this test is the snapshot of scuttled context. I'm not sure we want that.